packages/mongodb-rag-ingest/src/embed/updateEmbeddedContent.test.ts

```diff
@@ -1,7 +1,17 @@
-import { EmbeddedContentStore, EmbeddedContent, Page } from "mongodb-rag-core";
-import { updateEmbeddedContent } from "./updateEmbeddedContent";
+import {
+  EmbeddedContentStore,
+  EmbeddedContent,
+  Embedder,
+  Page,
+  PersistedPage,
+} from "mongodb-rag-core";
+import {
+  updateEmbeddedContent,
+  updateEmbeddedContentForPage,
+} from "./updateEmbeddedContent";
 import { persistPages } from "../pages";
 import { makeMockPageStore } from "../test/MockPageStore";
+import * as chunkPageModule from "./chunkPage";
```

Comment:
can you just import the function `chunkPage()`, and spy on that rather than do a * import?

Reply:
I cant seem to do that without having to use `jest.spyOn(require('./chunkPage'), 'chunkPage')`
which causes this eslint error: Require statement not part of import statement. 
If i change the import to `import { chunkPage } from './chunkPage'; `, the mocking doesn't work correctly and the test fails.

Reply:
ok lets leave it as is then